tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_camera_with_passcode.py

```diff
@@ -17,7 +17,27 @@ def setUp(self):
         # Turn off geolocation prompt
         self.apps.set_permission('Camera', 'geolocation', 'deny')
 
-        self.data_layer.set_setting('lockscreen.passcode-lock.code', self._input_passcode)
+        #set passcode-lock
+        SET_DIGEST_VALUE = 'lockscreen.passcode-lock.digest.value'
+        SET_DIGEST_SALT = 'lockscreen.passcode-lock.digest.salt'
+        SET_DIGEST_ITERATIONS = 'lockscreen.passcode-lock.digest.iterations'
+        SET_DIGEST_ALGORITHM = 'lockscreen.passcode-lock.digest.algorithm'
+
+        digestNums = [195, 174, 33, 98, 39, 43, 135, 112,
+                      126, 176, 82, 150, 236, 112, 87, 54,
+                      96, 60, 208, 18, 86, 178, 19, 20,
+                      129, 91, 168, 134, 241, 138, 59, 210]
+
+        Settings = {}
+        Settings[SET_DIGEST_VALUE] = digestNums # should be digest1337 = new Uint8Array(digestNums)
+        Settings[SET_DIGEST_SALT] = [4, 8, 15, 16, 23, 42, 108, 0] # should
+        # be uint8array
+        Settings[SET_DIGEST_ITERATIONS] = 1000
+        Settings[SET_DIGEST_ALGORITHM] = 'SHA-1'
+
+        for setting, value in Settings.iteritems():
+            self.data_layer.set_setting(setting, value)
+
```

Comment:
Yeah, I can write up a test class that contains an abstraction. That sounds like a good idea!
 Would a `PasscodeTest` class with a method called `setPasscodeTo1337()` work for you?
(Re-Implementing PBKDF2 for these tests doesn't sound ideal to me, so I'd prefer keeping the pre-computed values in here)